interfaces/dummy-oauth/dummy-oauth.yaml

```diff
@@ -79,3 +63,393 @@ paths:
           description: >-
             The request was not properly formed
       summary: Generate an access token
+  /fims/token:
+    post:
+      summary: Request an access token.
+      description: |
+        The primary endpoint for this authorization server.  Used to request an access token
+        suitable for authorizing data exchanges within the USS Network.
+
+        Implemented per https://tools.ietf.org/html/rfc6749#section-3.2 .
+      parameters:
+        - in: header
```

Comment:
This breaks API autogeneration (`make apis`) currently -- do we really need to require this input since the only thing we seem to do with it is echo its value in the response message?

Reply:
Yes. I had to make changes to openapi-to-go-server to generate the code related to headers. But, I copied only the POST /token changes to mock_uss. Also the types generation of models was not correct through openapi-to-go-server.
But seems like there are checks on the 3 go files in dummy-auth w.r.t openapi-to-go-server. I ll look into removing headers completely. Initially, we had thought of being able to at least print the faa uft headers for POST /token using dummy-oauth.

Reply:
If changes to openapi-to-go-server were needed to generate the code included in this PR, either those changes should be merged in a separate PR prior to having this PR reviewed, or else be included in this PR, or else remove the generated code from this PR.  I think the quickest path to booked success (merging PRs) would be to limit this PR solely to the changes in dummy-oauth.yaml (then the next PR would include both changes to openapi-to-go-server and autogeneration).

> Initially, we had thought of being able to at least print the faa uft headers for POST /token using dummy-oauth.

That should still probably be possible even if dummy-oauth.yaml doesn't specify the headers as required.